tests/water_fugacity_prefactor.prm

```diff
@@ -0,0 +1,123 @@
+# A test for the exact water fugacity prefactor
+# with the visco_plastic material model.
+# Following the flow law for wet diffusion from
+# Hirth & Kohlstaedt 2004 the exact viscosity is
+# calculated in python through:
+# import numpy as np
+# m = 3
+# d = 1e-3 m
+# E = 375e3 J/mol
+# V = 15e-6 m^3/mol
+# R = 8.3144621
+# P = 10e9 Pa
+# r = 0.7
+# T = 1673 K
+# A_diff = 2.5e7 / 1e6 / 1e6**m / 1e6**r 1/Pa^2/m^3/s
+
+
+# A_h2o = 26 / 1e6**r 1/Pa
+# E_h2o = 40e3 J/mol
+# V_h2o = 10e-6 m^3/mol
+
+# M_h2o = 18.01528
+# M_ol = 147.0027
+
+# wt_fraction_h2o = 0.01
+# wt_ol = 1 - wt_fraction_h2o
+# COH = (wt_fraction_h2o/M_h2o) / (wt_ol/M_ol) * 1e6
+
+# fh2o = COH / A_h2o * np.exp((E_h2o + P*V_h2o)/(R * T))
+# viscosity_analytic = 0.5 * A_diff**(-1) * d**(m) * np.exp((E + P*V) / R / T) * fh2o**(-r)
+# In the background composition, the viscosity should be equal to 2.766e19,
+# In the solid_2 composition, r=0.7 and the viscosity should be equal to 4.924e23
+# Which should result in an average model viscosity of 2.462e23.
+# The material model should return this viscosity.
+
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
+set Surface pressure                       = 10.e9
+
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = solid_2, bound_fluid
+end
+
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = if(y<=50e3, 1.0, 0.0); 0.01
+  end
+end
+
+# Model geometry (100x100 km, 10 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# No boundary-driven deformation, but the strain-rate used by the material
+# model is set in the material model section through the parameters
+# "Reference strain rate" and "Minimum strain rate".
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, top, left, right
+end
+
+# Initial temperature conditions
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 1673
+  end
+end
+
+# Material model (values for background material)
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+    # dislocation creep parameters
+    # These values produce such a sufficiently high viscosity that the
```

Comment:
I think this documentation is no longer relevant. Can you update to describe what is happening in this specific model (prefactor multiplication, etc).